pdr_backend/pdr_dashboard/dash_components/plots.py

```diff
@@ -99,16 +103,25 @@ def _empty_profit_scatter() -> List[go.Scatter]:
     return [go.Scatter(x=[], y=[], mode="lines", name="profit")]
 
 
+def _empty_costs_bar() -> List[go.Bar]:
+    return [go.Scatter(x=[], y=[], name="costs", width=5)]
+
+
 def _empty_stakes_bar() -> List[go.Bar]:
     return [go.Bar(x=[], y=[], name="stakes", width=5)]
 
 
-def _empty_trio() -> Tuple[go.Figure, go.Figure, go.Figure]:
-    return _empty_accuracy_scatter(), _empty_profit_scatter(), _empty_stakes_bar()
+def _empty_trio() -> Tuple[go.Figure, go.Figure, go.Figure, go.Figure]:
```

Comment:
it's not a trio anymore, though :)

Reply:
i have no idea what a trio is (i do) but the naming has to change if the parameters ever do

kiss... just name it tuple